src/frontend/next/src/styles/globals.css

```diff
@@ -4,6 +4,8 @@
 /* Import highlight.js */
 @import 'highlight.js/styles/github.css';
 
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700');
+
```

Comment:
We can't do this. We need to have all the assets in our project. 
I believe this font was already downloaded because MUI needs it. 

@tonyvugithub, where are we storing our fonts?

Reply:
Yeah Idk why the 'fontFamily: roboto' inside Banner.tsx doesn't work, so I just tried to put it global temporary, but obviously it affects other components